action.yaml

```diff
@@ -1,70 +1,69 @@
-name: 'Get deployment information'
+name: "Get deployment information"
 description: "Get the information required to build and deploy an Octue service."
-author: 'cortadocodes'
+author: "cortadocodes"
 #branding:
 #  icon: git-pull-request
 #  color: green
 inputs:
   gcp_project_name:
-    description: ''
+    description: "The name of the Google Cloud project being deployed to."
     required: true
   gcp_project_number:
-    description: ''
+    description: "The number of the Google Cloud project being deployed to."
     required: true
   gcp_region:
-    description: ''
+    description: "The Google Cloud region being deployed to."
     required: true
   gcp_resource_affix:
-    description: ''
+    description: ""
+    required: true
+  gcp_service_name:
+    description: "The name of the service being deployed."
     required: true
   gcp_environment:
-    description: ''
+    description: ""
```

Comment:
I'm not sure what this is @thclark - the branch name?

Reply:
Not necessarily; it's usually either "production" or "staging" because we have one environment that has all the different branches in staging, and one environment that has only one branch (main) in production.

But I haven't figured out a good strategy around this area yet, which is why it's ill defined.

I might request some changes here to actually set it based on the branch tag internal to this action but let's leave it like this for now as I try to consolidate architectures.

Reply:
Ok I'll add a description reflecting what it is currently for now